libsolidity/interface/CompilerStack.cpp

```diff
@@ -594,8 +594,13 @@ bool CompilerStack::checkLibraryNameClashes()
 string CompilerStack::absolutePath(string const& _path, string const& _reference) const
 {
 	// Anything that does not start with `.` is an absolute path.
-	if (_path.empty() || _path.front() != '.')
+	size_t slashPos { _path.find_first_of('/') };
```

Comment:
`boost::filesystem::path` has no `[]` operator.  I haven't found anything useful in the library.